tests/test_nce.py

```diff
@@ -0,0 +1,20 @@
+"""
+Test functions for the NCE metric (src/locomoset/metrics/nce.py)
+"""
+
+import numpy as np
+import pytest
+
+from locomoset.metrics.nce import nce
+
+
+def test_nce_perfect_features(dummy_labels):
+    """Test that NCE returns approximately zero for perfect features"""
+    assert nce(dummy_labels, dummy_labels) == pytest.approx(0)
+
+
+def test_nce_random_features(dummy_labels, dummy_n_classes, test_n_samples):
+    """Test that NCE returns a negative value"""
+    rng = np.random.default_rng(43)
```

Comment:
would be better to use the `test_seed` from `conftest` here, rather than using 43

Reply:
Maybe I've misunderstood how the `rng` generator works, but I wanted to have distinct value from the dummy labels, but using the same method, hence using an alternate seed. Should I make an `alt_test_seed` in `conftest`?

Reply:
ah ok makes sense, maybe add a short comment to say that

Reply:
done